.github/workflows/build-conditions-konto.yml

```diff
@@ -0,0 +1,129 @@
+name: Workflow which determines, what should be built on konto.bratislava.sk projects
+on:
+  workflow_call:
+    outputs:
+      strapi:
+        description: "Returns true if strapi can be built"
+        value: ${{ jobs.conditions.outputs.strapi }}
+      next:
+        description: "Returns true if next can be built"
+        value: ${{ jobs.conditions.outputs.next }}
+      nest-forms-backend:
+        description: "Returns true if nest-forms-backend can be built"
+        value: ${{ jobs.conditions.outputs.nest-forms-backend }}
+jobs:
+  changes:
+    name: Checking for directory changes in repository
+    runs-on: ubuntu-latest
+    outputs:
+      strapi: ${{ steps.filter.outputs.strapi }}
+      next: ${{ steps.filter.outputs.next }}
+      nest-forms-backend: ${{ steps.filter.outputs.nest-forms-backend }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+
+      - name: Pipelines Version
+        run: |
+          echo "Pipelines version: 2.2.1"
+
+      - uses: dorny/paths-filter@v3
+        id: filter
+        with:
+          filters: |
+            strapi:
+              - 'strapi/**'
+            next:
+              - 'next/**'
+            nest-forms-backend:
+              - 'nest-forms-backend/**'
+  conditions:
+    name: Checking build conditions
+    needs: changes
+    runs-on: ubuntu-latest
+    env:
+      COUNT: 0
+    outputs:
+      strapi: ${{ steps.strapi.outputs.condition }}
+      next: ${{ steps.next.outputs.condition }}
+      nest-forms-backend: ${{ steps.nest-forms-backend.outputs.condition }}
+    steps:
+      - name: Print pipeline comment
+        if: github.event_name == 'pull_request'
+        uses: marocchino/sticky-pull-request-comment@v2.9.0
+        with:
+          header: Build Conditions
+          recreate: true
+          message: |
+            <b> Test build pipeline info :rocket: </b><br/>
+            Changes in the code and tag info: <br/>
+            :arrow_right: Changes in strapi: <b>${{ needs.changes.outputs.strapi }}</b><br/>
+            :arrow_right: Changes in next: <b>${{ needs.changes.outputs.next }}</b><br/>
+            :arrow_right: Changes in nest-forms-backend: <b>${{ needs.changes.outputs.nest-forms-backend }}</b><br/>
+            We are going to build :ship:
+
+      - id: nest-forms-backend
+        name: Check for build nest-forms-backend
+        if: needs.changes.outputs.nest-forms-backend == 'true'
+        run: |
+          echo "COUNT=$(($COUNT+1))" >> $GITHUB_ENV
+          echo "condition=true" >> $GITHUB_OUTPUT
```

Comment:
Isn't this just relabeling of `needs.changes.outputs.nest-forms-backend` to `needs.conditions.output.nest-forms-backend.condition`. Can't we drop this step and just use the output of `changes`? 

Same goes for all other steps in this job.